test-clients/src/main/resource/contract/bytecodes/PayableConstructor.bin

```diff
@@ -1 +1 @@
-6080604052603e8060116000396000f3fe6080604052600080fdfea265627a7a723158209dcac4560f0f51610e07ac469a3401491cfed6040caf961950f8964fe5ca3fe264736f6c634300050b0032
\ No newline at end of file
+6080604052603e8060116000396000f3fe6080604052600080fdfea265627a7a723158204be1c10b97f2e7e00079115200c21ebf2c15c8eda66596685150cd6f5356464664736f6c63430005110032
```

Comment:
If the PayableConstructor contract is changed, should we also update PayableConstructor.sol?

Reply:
I actually copied the code of `PayableConstructor.sol` and compiled it in remix, so this bytecode is actually the re compiled version of the solidity code in the `PayableConstructor.sol`